src/map.js

```diff
@@ -523,17 +520,19 @@ var map = function (arg) {
    * Set center of the map to the given geographic coordinates, or get the
    * current center.  Uses bare objects {x: 0, y: 0}.
    *
-   * @param {Object} coordinates
-   * @param {string|geo.transform} [gcs] undefined to use the interface gcs,
-   *    null to use the map gcs, or any other transform.  If setting the
-   *    center, they are converted from this gcs to the map projection.  The
-   *    returned center are converted from the map projection to this gcs.
-   * @param {boolean} ignoreDiscreteZoom if true, ignore the discreteZoom
+   * @param {geo.geoPosition} coordinates
+   * @param {string|geo.transform|null} [gcs] undefined to use the interface
+   *    gcs, null to use the map gcs, or any other transform.  If setting the
+   *    center, it is converted from this gcs to the map projection.  The
+   *    returned center is converted from the map projection to this gcs.
+   * @param {boolean} [ignoreDiscreteZoom] if true, ignore the discreteZoom
    *    option when determining the new view.
-   * @param {boolean} ignoreClampBounds if true or 'limited', ignore the
-   *    clampBoundsX options (up to a point, see fix_bounds) when determining
-   *    the new view.
-   * @returns {Object|geo.map}
+   * @param {boolean|'limited'} [ignoreClampBounds] if true ignore the
+   *    clampBounds* options when determining the new view.  When 'limited',
```

Comment:
shortened form of `clampBoundsX and clampBoundsY options`.  Obviously not clear enough, so I shouldn't use the shortened form